testing/unit_tests/test_loader.py

```diff
@@ -244,14 +243,51 @@ def test_get_target_filename_list(loader_parameters, model_parameters, transform
     bids_df = imed_loader_utils.BidsDataframe(loader_parameters, __tmp_dir__, derivatives=True)
     data_lst = ['sub-rat3_ses-01_sample-data9_SEM.png']
     test_ds = imed_loader.load_dataset(bids_df,
-                                  **{**loader_parameters, **{'data_list': data_lst,
-                                                             'transforms_params': transform_parameters,
-                                                             'dataset_type': 'training'}})
-
+                                       **{**loader_parameters, **{'data_list': data_lst,
+                                                                  'transforms_params': transform_parameters,
+                                                                  'dataset_type': 'training'}})
     target_filename = test_ds.filename_pairs[0][1]
     
     assert len(target_filename) == len(loader_parameters["target_suffix"])
 
 
+@pytest.mark.parametrize('loader_parameters', [{
```

Comment:
Reviewed but not tested, I trust you on this one @mariehbourget, except if you think important that someone else test it, 'd be happy to: lemme know :-)